plenum/test/replica/test_buffers_cleaning.py

```diff
@@ -0,0 +1,63 @@
+from plenum.server.replica import Replica
+
+
+class FakeNode():
```

Comment:
Because node can makes some additional initialization steps, creates files etc.
But this test checks quite specific replica functionality, so that's why I abstracted it from other modules.